submodules/eps/__init__.py

```diff
@@ -137,9 +138,9 @@ def led_on_off() -> None:  # TODO: Delete this method, from eps testing with fak
     looptime = 20  # FIXME: Was 30
     while True:
         pin_on('aprs')
-        time.sleep(looptime)
+        time.sleep(config['eps']['looptime'])
```

Comment:
what is the purpose of `looptime`

Reply:
^^^

Reply:
idk but it's from last december so its outdated plus jason said he's updating eps so it's probably not important